src/views/CreateTransformer.vue

```diff
@@ -0,0 +1,175 @@
+<template>
+  <main v-if="formState === 0" @keydown.enter="nextStep">
+    <ViewHeader title="How should the new transformer be called?" :backLink="{ link: { name: 'transformers' } }"/>
+    <Input :value.sync="newTransformer.name" name="Name" placeholder="Filter for today" :autofocus="true"/>
+    <Button text="Continue" :onClick="nextStep"/>
```

Comment:
You can continue, even if the `name` is empty.

Reply:
This seems to be a general issue...

Reply:
it is a general issue. But at least you get an error messeage when you try to save!

Reply:
Maybe we should open a Bug ticket for this?

Reply:
Sorry, my fault